### ISSUE TITLE
Tibia 11

### ISSUE THREAD

**Issue by snax4a** [Mytherin/Tibialyzer#348]
Does it work with newest tibia client?

**Comment by benjaminkomen** [comment]
I don't think so. This project was sort of abandoned when CipSoft introduced an external bot-detection tool (BattleEye I believe). Also, CipSoft incorporated a lot of the features Tibialyzer provided, directly into the Tibia client.
